src/main/webapp/js/finscholar/init.js

```diff
@@ -17,14 +17,33 @@
 goog.module('finscholar');
 
 const GoogDom = goog.require('goog.dom');
+const {CollegePageView} = goog.require('finscholar.collegepageview');
 const {HomePageController} = goog.require('finscholar.homepagecontroller');
 
-const init = () => {
-  console.log('TODO: implement this.');
+/**
+ * Initialize the main web page. 
+ */
+const init = async () => {
   const homeController = new HomePageController();
-  console.log(GoogDom);
   GoogDom.getElement('main').innerHTML = homeController.content;
 
+  // Render the college page by loading in the data and rendering the 
+  // associated soy template.
+  try {
+    await renderCollege(GoogDom.getElement('college'));
+  } catch(err) {
+    alert('Failed to initialize page. College page could not be rendered.');
+    console.log(err);
+  }
+};
+
+/**
+ * Render the college page.
+ * @param {Element} element - The DOM element to attach the college page to.
+ */
+const renderCollege = async (element) => {
```

Comment:
Should this function be in init.js? For clarity you can move it to a module related to college object.